packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -0,0 +1,141 @@
+import type {
+  GetMembersParams,
+  MembershipRole,
+  OrganizationInvitationJSON,
+  OrganizationJSON,
+  OrganizationMembershipJSON,
+  OrganizationResource,
+} from '@clerk/types';
+import { unixEpochToDate } from 'utils/date';
+
+import {
+  BaseResource,
+  OrganizationInvitation,
+  OrganizationMembership,
+} from './internal';
+
+export class Organization extends BaseResource implements OrganizationResource {
+  id!: string;
+  name!: string;
+  role!: MembershipRole;
+  instanceId!: string;
+  createdBy!: string;
+  createdAt!: Date;
+  updatedAt!: Date;
+
+  constructor(data: OrganizationJSON) {
+    super();
+    this.fromJSON(data);
+  }
+
+  static async create(name: string): Promise<OrganizationResource> {
```

Comment:
❓   IIRC, all resources use the `this.basePost | this.baseGet | ...` methods that retrieve a json and return the instantiated resource. Why did we decide to use the static `BaseResource._fetch` here?

To be honest, I'm not a huge fan of the `this.basePost` approach, but if we don't have any specific reasons to do otherwise, I think it'd be better to remain consistent and tackle them all at once in a different PR

Reply:
@nikosdouvlis 
Discussed offline.